### ISSUE TITLE
Error praising package

### ISSUE THREAD

**Issue by iCeY3** [kaungsatthe1n/Tako-Play#6]
As title says cant install

**Comment by iCeY3** [comment]
![Uploading Screenshot_20220121-095420_Package installer.jpg…]()

**Comment by kaungsatthe1n** [comment]
@iCeY3 Please make sure to Allow Unknown Resources in your android setting.

**Comment by kaungsatthe1n** [comment]
@iCeY3 Please make sure to Allow Unknown Resources in your android setting.

**Comment by iCeY3** [comment]
> @iCeY3 Please make sure to Allow Unknown Resources in your android setting.

Everything is enabled i have many apps...
Im running andriod 12 galaxsy s21 ultra

**Comment by Fruelund** [comment]
Got the same issue. ![Screenshot_20220124-075453_Package installer.jpg](https://user-images.githubusercontent.com/90288058/150735828-18821930-eeaa-4750-8c77-e133549f1ede.jpg)

**Comment by iCeY3** [comment]
@iCeY3 still same issue with the new update

**Comment by kaungsatthe1n** [comment]
@iCeY3 https://github.com/kaungsatthe1n/Tako-AnimeTracker 
try this app on your device.

**Comment by iCeY3** [comment]
@kaungsatthe1n it installed

**Comment by Fruelund** [comment]
Same.

**Comment by kaungsatthe1n** [comment]
@Fruelund  @iCeY3  
Alright ..

**Comment by kaungsatthe1n** [comment]
@Fruelund @iCeY3 
Please download the latest release..

**Comment by Fruelund** [comment]
Done. Thanks.

Do you plan on making it work with AniList?

**Comment by ITADORI-15** [comment]
Ya anilist update would be sikeee

**Comment by kaungsatthe1n** [comment]
Yea.. I will consider guys ..

**Comment by ITADORI-15** [comment]
Bro My device is Samsung M30
And it's working perfectly fine from the beginning

**Comment by kaungsatthe1n** [comment]
what happened next ..

**Comment by kaungsatthe1n** [comment]
@ITADORI-15  Yes, Because it depends on android sdk version and your android version is below 31.
That's why Android phone with sdk version 31 and above 31 are getting error.

**Comment by ITADORI-15** [comment]
Yeah guess so
